src/main/java/slimeknights/tconstruct/library/registration/RecipeSerializerDeferredRegister.java

```diff
@@ -0,0 +1,34 @@
+package slimeknights.tconstruct.library.registration;
+
+import net.minecraft.item.crafting.IRecipeSerializer;
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.registries.DeferredRegister;
+import net.minecraftforge.registries.ForgeRegistries;
+import slimeknights.tconstruct.library.registration.object.RecipeSerializerObject;
+
+import java.util.function.Supplier;
+
+public class RecipeSerializerDeferredRegister extends RegisterWrapper<IRecipeSerializer<?>> {
```

Comment:
Delete this, all it does is apply the wrapper, which is unneeded wrapping. Use the base deferred register

Reply:
Ah yeah. There was no base defferred reg in tree when I wrote this. Actually,
I think there still isn't; isn't that in your partbuilder branch?

Reply:
There was always a base deferred register. Forge is the base deferred register.